Formula/gopass.rb

```diff
@@ -48,7 +53,7 @@ def install
     ensure
       system Formula["gnupg"].opt_bin/"gpgconf", "--kill", "gpg-agent"
       system Formula["gnupg"].opt_bin/"gpgconf", "--homedir", "keyrings/live",
-                                                 "--kill", "gpg-agent"
+             "--kill", "gpg-agent"
```

Comment:
Let's not touch this line.

Please squash this change into your existing commit and use the commit message
```
gopass: add completions and manpage
```
See the commit style guide for more details.